src/components/FeatureNotification/OrganizationNotification.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box } from '@mui/material';
+import { APP_PATHS } from 'src/constants';
+import strings from 'src/strings';
+import { Notification } from 'src/types/Notifications';
+import TextWithLink from 'src/components/common/TextWithLink';
+import { useOrganization } from 'src/providers';
+import { getTodaysDateFormatted } from '@terraware/web-components/utils';
+
+export default function OrganizationNotification(orgTimeZone?: string): Notification {
+  const { selectedOrganization } = useOrganization();
+
```

Comment:
Move the logic for this notification display into this component... if we don't need to show the notification we can return null in this component...
The mark read logic can be inside here as well?